framework/src/bundle/BundleUtils.h

```diff
@@ -34,6 +35,15 @@ void* GetExecutableHandle();
 
 // returns the address of the symbol in library libHandle
 void* GetSymbol(void* libHandle, const char* symbol);
+
+template<typename T> void GetSymbol(std::function<T>& fptr,
```

Comment:
This is used in on call site which sets the value for an instance of BundlePrivate. All of the call sites are within BundlePrivate::Start0 and look like this:

      BundleUtils::GetSymbol(SetBundleContext, libHandle, set_bundle_context_func);

Perhaps a change in name to "LookupFunction" and a rename of the parameter to outFuncPtr or something like that?

If I make the function pointer a return value, there will be compilation problems because of the other "GetSymbol" function which would have the same argument list but a different return value (a void* instead of the std::function).